jest.config.ts

```
@@ -159,9 +159,10 @@ export default {
   // ],
 
   // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
-  // testPathIgnorePatterns: [
-  //   "/node_modules/"
-  // ],
+  testPathIgnorePatterns: [
+    "/node_modules/",
```

ESLint

[error] Replace `"/node_modules/"` with `'/node_modules/'`

(prettier/prettier)